CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.18.0] Q1 2024
+- added ability to specify authentication provider and environment with
+  configuration files (default to prod environment)
+- removed bollard logging setup
+- bumped version of toml crate
+- fixed error where bmap files would not match to images after omnect-cli operations
```

Comment:
- changed default compression rate to 4
- integration test:
  - check if image is changed or unchanged after a command was run
  - check generated bmap file